images/kwok/build.sh

```diff
@@ -139,6 +145,17 @@ function main() {
       "--platform=${platform}"
     )
   done
+
+  if [[ "${BUILDER}" == "nerdctl" ]]; then
+    build_with_nerdctl "${extra_args[@]}"
+  else
+    build_with_docker "${extra_args[@]}"
+  fi
+}
+
+function build_with_docker() {
+  local extra_args
+  extra_args=("$@")
```

Comment:
```suggestion
  local name="$1"
  shift
  local extra_args
  extra_args=("$@")
```

Reply:
the same as previous one

> we can not use `${image}:${tag}` as a push image directly, because multiple images is supported.
>
> I have updated to a declare array to support this.
>
> also, nerdctl is not support pushing multiple images in the same time, a loop is added